tiledb/ml/models/sklearn.py

```diff
@@ -85,16 +83,15 @@ def _create_array(self):
 
         # In case we are on TileDB-Cloud we have to update model array's file properties
         if self.namespace:
-            tiledb.cloud.array.update_file_properties(
-                uri=self.uri,
-                file_type=FILETYPE_ML_MODEL,
-                file_properties={
-                    FilePropertyName_ML_FRAMEWORK: "SKLEARN",
-                    FilePropertyName_STAGE: "STAGING",
-                    FilePropertyName_PYTHON_VERSION: platform.python_version(),
-                    FilePropertyName_ML_FRAMEWORK_VERSION: sklearn.__version__,
-                },
-            )
+            from tiledb.ml._cloud_utils import update_file_properties
+
+            file_properties = {
+                FilePropertyName_ML_FRAMEWORK: "SKLEARN",
+                FilePropertyName_STAGE: "STAGING",
+                FilePropertyName_PYTHON_VERSION: platform.python_version(),
+                FilePropertyName_ML_FRAMEWORK_VERSION: sklearn.__version__,
```

Comment:
(unrelated to this PR) The `FileProperty*` constants could be packaged as an [enum](https://docs.python.org/3/library/enum.html)